tests/math_builtin_api/math_builtin.h

```diff
@@ -130,16 +131,28 @@ bool verify(sycl_cts::util::logger &log, sycl::vec<T, N> a,
   return true;
 }
 
+template <typename T, size_t N>
+bool verify(sycl_cts::util::logger &log, sycl::marray<T, N> a,
+            sycl_cts::resultRef<sycl::marray<T, N>> r, int accuracy,
+            const std::string &comment) {
+  sycl::marray<T, N> b = r.res;
+  for (size_t i = 0; i < N; i++)
+    if (r.undefined.find(i) == r.undefined.end() &&
+        !verify(log, a[i], b[i], accuracy, comment))
+      return false;
+  return true;
+}
+
 template <typename T>
 bool verify(sycl_cts::util::logger &log, T a, T b, int accuracy,
-            std::string comment) {
+            const std::string &comment) {
   return verify(log, a, sycl_cts::resultRef<T>(b), accuracy, comment);
 }
 
 template <int N, typename returnT, typename funT>
 void check_function(sycl_cts::util::logger &log, funT fun,
                     sycl_cts::resultRef<returnT> ref, int accuracy = 0,
-                    std::string comment = "") {
+                    const std::string &comment = "") {
```

Comment:
While you are here :-)
```suggestion
                    const std::string &comment = {}) {
```
You do not want every time to measure this empty C string and then construct a `std::string` from it...

Reply:
Done.